rmgpy/cantherm/gaussian.py

```diff
@@ -156,12 +156,14 @@ def loadGeometry(self):
                 mass[i] = 31.97207100
             elif number[i] == 17:
                 mass[i] = 35.4527
+            elif number[i] == 53:
+                mass[i] = 126.90447
             else:
                 raise NotImplementedError('Atomic number {0:d} not yet supported in loadGeometry().'.format(number[i]))
         
         return coord, number, mass
 
-    def loadConformer(self, symmetry=None, spinMultiplicity=None, opticalIsomers=1):
+    def loadConformer(self, symmetry=None, spinMultiplicity=None, opticalIsomers=1, symfromlog=None):
```

Comment:
The gaussianTest currently fails since it doesn't pass this argument, and symmetry isn't being read.
Could you change `conformer = log.loadConformer()` into `conformer = log.loadConformer(symfromlog=True)` in [gaussianTest.py](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/cantherm/gaussianTest.py#L81)?
While we're there, and if you don't mind, could you also add this parameter to [QChem](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/cantherm/qchem.py#L179) (and relevant tests)?